api/OpenAI.net8.0.cs

```diff
@@ -5593,6 +5656,7 @@ public enum MessageStatus {
     [Experimental("OPENAI001")]
     public class OpenAIResponse : IJsonModel<OpenAIResponse>, IPersistableModel<OpenAIResponse> {
```

Comment:
The `OpenAIResponse` class can be removed entirely in favor of `ResponseResult`.